src/main/webapp/pages/index.html

```diff
@@ -45,12 +45,15 @@ <h4 class="pull-left" th:text="#{announcement.title}">_New Announcements_</h4>
         </div>
       </div>
       <div class="panel-body">
-        <div class="form-group">
-          <announcements data:url="@{announcements/user/unread}"></announcements>
+        <div id="dashboard-root">
+          <!-- Contents rendered from DashboardPage.jsx -->
```

Comment:
This comment is old now.

Reply:
Please see 97ea7082c57aa57f303552bac452deb1d4defaa9.